src/content/docs/synthetics/synthetic-monitoring/administration/user-roles-synthetic-monitoring.mdx

```diff
@@ -12,24 +12,25 @@ redirects:
 
 Access to synthetic monitoring features varies whether a user is on the newer [New Relic One user model](#new-pricing) or our [original user model](#original-pricing).
 
-For an explanation of user models and how they relate to our pricing models, see [Overview of pricing and user changes](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#convert-users).
+For an explanation of user models, see [User models](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models).
 
 ## Users on original user model [#original-pricing]
 
-For users on our original user model, access will vary depending on whether you have default roles or custom roles:
+Use of the synthetic monitoring UI requires a user type of [full platform user](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) user type. 
```

Comment:
A little weird. I'd use "Use of the synthetic monitoring UI requires a [full platform user](https://github.com/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) user type."